### PR TITLE
fix wrong tab completion

### PR DESCRIPTION
**Pull request by Mashimiao**
Signed-off-by: Ma Shimiao mashimiao.fnst@cn.fujitsu.com

Subcomand insert does not exist, so delete it from tab  completion. 
Add subcomand logout to tab completion.

### CODE REVIEW

File: contrib/completion/zsh/_docker [hunk 1]
```diff
@@ -282,15 +277,10 @@ __docker_subcommand () {
             ;;
         (info)
             ;;
-        (import)
+        (inspect)
             _arguments \
-                ':URL:(- http:// file://)' \
-                ':repository:__docker_repositories_with_tags'
```
Comment (icecrime) [inline contrib/completion/zsh/_docker]: Why drop this line?

**Comment by icecrime** [comment]
I thought fish completion was generated by a script (cf. comments at the top of the `.fish` file), how did it ended up being wrong?

**Comment by Mashimiao** [comment]
I think the script it is a little old. It supports docker1.3, while docker's mainline has changed a lot from 1.3 to 1.5

**Comment by icecrime** [comment]
LGTM

**Comment by jfrazelle** [comment]
LGTM ping @tianon

**Comment by jfrazelle** [comment]
ping @albers for verification too :D

**Comment by albers** [comment]
For the bash completion, there was kind of a race condition with https://github.com/docker/docker/commit/04ca2c1f2bcf9251689dc346baed6715fb99fb0e#diff-754420541b12dbcbbea00825942a4635L946. As git sees no problem wrt merging, LGTM.
I'm not familiar with the zsh completion, but it looks reasonable. :+1:

**Comment by albers** [comment]
Thanks @Mashimiao for your contribution!

**Pull request merged**